Add a note on service account usage in Airflow

ptiurin opened the pull request:
FIR-20509

ptiurin committed:
Add a note on service account usage in Airflow

FIR-20509

molly-firebolt commented:
Hey @ptiurin is this applicable for FB2.0 only, or FB1.0 as well? Also, the ticket linked needs to be checked as "Docs required" and moved back to Rolling out status until this is published - I will close it at that point, but it will not get a fix version linked unless it's in Rolling out.

ptiurin commented:
> Hey @ptiurin is this applicable for FB2.0 only, or FB1.0 as well? Also, the ticket linked needs to be checked as "Docs required" and moved back to Rolling out status until this is published - I will close it at that point, but it will not get a fix version linked unless it's in Rolling out.

Hi @molly-firebolt , this is for FB 1.0 only, FB 2.0 changes are not there yet. 
I've updated the ticket.

molly-firebolt committed:
Update airflow.md
